pyseir/icu/utils.py

```diff
@@ -0,0 +1,36 @@
+from libs.datasets import combined_datasets
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets.dataset_utils import AggregationLevel
+import pandas as pd
+import us
+
+
+def _cases_quantile_range(x):
+    """Compute the quantile range for the cumulative case data.  We sometimes get single points of
+    a timeseries that are obviously wrong. Taking the quantile range is less sensitive to outliers
+    than either just "last minus first" or "maximum minus minimum"
+    """
+    QUANTILES = [0.05, 0.95]  # We sometimes get single point obviously wrong input.
+    # Quantile range is less sensitive than max-min or last-first.
+    lower, upper = x[CommonFields.CASES].quantile(q=QUANTILES).values
+    return upper - lower
+
+
+def calculate_case_based_weights():
```

Comment:
Can you include a cli command to actually run the output and update the files? makes maintenance a lot easier for others

Reply:
I added it in the if main loop so a `python pyseir/icu/utils.py` call will do the job. Once we switch PySEIR out I'm happy to find a better place for it.